doc/integrator/backend_qgis.rst

```diff
@@ -57,45 +58,64 @@ Remove the mapserver folder:
 
    git rm mapserver
 
-Connect to Postgres from QGIS desktop
--------------------------------------
+QGIS Desktop configuration
+--------------------------
 
-This chapter is subject to change when the QGIS plugin is available.
+OWS configuration
+~~~~~~~~~~~~~~~~~
 
-Throw a tunnel
-~~~~~~~~~~~~~~
+You should setup your OWS service in the QGIS project properties in the OWS
+tab.
 
-The first solution is to create a tunnel with ssh:
+You should take care to **uncheck** the checkbox *User Layer ids as names*. If
+this checkbox is enabled you will have unreliabel layers name.
 
-.. prompt:: bash
+You should **check** the checkbox *Add geometry to feature response* in order
+to make working the WMS GetFeatureInfo correctly.
+
+In the *WFS capabilities* section, you should check the layers that need to be
+available in the WFS service.
+
+Finally, your QGIS project layers CRS should all be in the same CSR. This is subject to
+change.
+
+Connect to Postgres database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section is subject to change when the QGIS plugin is available.
+
+* Throw a tunnel
```

Comment:
Proposed structure:
```rst
=========================================
Specific configuration for QGIS mapserver
=========================================

Viewer configuration
====================

Apache configuration
====================

Application configuration
=========================

Cleanup
=======

QGIS Desktop configuration
==========================

OWS configuration
-----------------

Connect to Postgres database
----------------------------

Throw a tunnel
~ ~~~~~~~~~~~~~

Directly on the server
~ ~~~~~~~~~~~~~~~~~~~~~

Deploy notes
============
```

I add some spaces to make it working in the comment :-)